File: problems/1351 - Count Negative Numbers in a Sorted Matrix/Go/Solution.go
func countNegatives(grid [][]int) int {
    m,n := len(grid), len(grid[0])
    res := 0

    for i := m - 1; i >= 0; i-- {
        sort.Ints(grid[i])

        for j := 0; j <= n; j++ {
            if grid[i][n-1] < 0 {
                res = res + n
                break;
            }

            if grid[i][j] < 0 {
                res++
            } else {
                break
            }
        } 
    }

    return res
}

File: problems/1351 - Count Negative Numbers in a Sorted Matrix/PHP/Solution.php
<?php

class Solution {

    /**
     * @param Integer[][] $grid
     * @return Integer
     */
    function countNegatives($grid) {
        $m = count($grid);
        $n = count($grid[0]);
        $res = 0;

        for ($i = $m - 1; $i >= 0; $i--) {
            sort($grid[$i]);

            for ($j = 0; $j <= $n; $j++) {
                if ($grid[$i][$n-1] < 0) {
                    $res = $res + $n;
                    break;
                }

                if ($grid[$i][$j] < 0) {
                    $res++;
                } else {
                    break;
                }
            }
        }

        return $res;
    }
}